wagtailio/project_styleguide/templates/patterns/base_page.html

```diff
@@ -1,4 +1,6 @@
-{% load static wagtailuserbar wagtailcore_tags wagtailimages_tags wagtail_ab_testing_tags wagtailio_utils manifest %}
+{% extends "patterns/base.html" %}
+{% load static wagtailcore_tags wagtailimages_tags wagtail_ab_testing_tags wagtailio_utils manifest %}
```

Comment:
`manifest` can be removed now the css and js files have been moved

Reply:
:+1: